### PR TITLE
Added new Reusable Button

### PR DESCRIPTION
**Pull request by ehteshaxm**

### CODE REVIEW

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]
Please take a look at how other components have been implemented. Your component does not build neither have you provided an example implementation in the example directory.

You can read the README.md of this repo for further clarification. If you have any more questions you can ask them on [discord](https://discord.gg/crZWFkXe)

File: example/src/components/Home.js [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
           </Link>
-          <Link to="/register" className="component">
+          <Link to='/register' className='component'>
             <h2>Register</h2>
           </Link>
-          <Link to="/product_card" className="component" >
+          <Link to='/product_card' className='component'>
             <h2>Product card</h2>
           </Link>
           {/* Add new Link Here for your new component */}
```
Comment (Ovenoboyo) [inline example/src/components/Home.js:27]: You're supposed to add your component here. Please check how other components have been implemented

File: example/src/components/button/button.module.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+.btn {
```
Comment (Ovenoboyo) [inline example/src/components/button/button.module.css:1]: unnecessary file

File: example/src/components/button/index.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import React from 'react'
```
Comment (Ovenoboyo) [inline example/src/components/button/index.js:1]: unnecessary file

File: example/src/components/Home.js (hunk 1 above)
Comment (Ovenoboyo) [reply example/src/components/Home.js:27]: Check App.js in src/example for more details

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]

File: example/src/components/Home.js [hunk 4]
```diff
@@ -27,6 +28,7 @@ const Home = () => {
...
         </div>
       </div>
+      <Button />
```
Comment (Ovenoboyo) [inline example/src/components/Home.js]: Please dont unnecessarily add your component in Home.js

**Review by Ovenoboyo — CHANGES_REQUESTED** [review]

**Comment by ehteshaxm** [comment]
I am getting an error that my component is not being imported from denezix-react in App.js inside example

**Comment by Ovenoboyo** [comment]
Can you show us what you've tried?

> I am getting an error that my component is not being imported from denezix-react in App.js inside example

**Comment by ehteshaxm** [comment]
I compiled again its working now.....

**Comment by ehteshaxm** [comment]
Please End this request I created a new one

**Pull request closed without merging**